Gemfile

```diff
@@ -23,6 +23,7 @@ gem "prawn"
 gem "puma"
 gem "rails", "~> 5.1"
 gem "responders"
+gem "ruby-filemagic"
```

Comment:
Per the note in the commit - don't forget to run `brew install libmagic` for this one.

Reply:
Do we need to add a buildpack for that to our app.json, staging, and production environments?

Reply:
I'll have to double-check whether this is in the default ruby buildpack but if not, I found this:

`heroku buildpacks:add --index 1 https://github.com/NetsoftHoldings/heroku-buildpack-magic.git`

Reply:
OK, let's make sure we test this in a review app before merging it then, and then add this buildpack to `app.json` if it is necessary.

Reply:
![michigan-benefits_ _pipeline___heroku](https://user-images.githubusercontent.com/28194/30489769-2173ae92-9a06-11e7-8c79-0170a454f01f.jpg)

All good on that front.